From 34d765e736e015f5c4790db2c0a1d3ddbd4bd3e9 Mon Sep 17 00:00:00 2001
From: Junio C Hamano <gitster@pobox.com>
Date: Fri, 23 Jun 2023 16:34:40 -0700
Subject: [PATCH 1659/2871] t6406: skip "external merge driver getting killed
 by a signal" test on Windows

The run_command() on the platform does not seem to report death by
signal as the caller expects.  For now, skip the test on Windows.

Signed-off-by: Junio C Hamano <gitster@pobox.com>
---
 t/t6406-merge-attr.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/t6406-merge-attr.sh b/t/t6406-merge-attr.sh
index b50aedbc00..9677180a5b 100755
--- a/t/t6406-merge-attr.sh
+++ b/t/t6406-merge-attr.sh
@@ -168,7 +168,7 @@ test_expect_success 'custom merge backend' '
 	rm -f $o $a $b
 '
 
-test_expect_success 'custom merge driver that is killed with a signal' '
+test_expect_success !WINDOWS 'custom merge driver that is killed with a signal' '
 	test_when_finished "rm -f output please-abort" &&
 
 	git reset --hard anchor &&
-- 
2.39.3 (Apple Git-145)

